webapp-django/crashstats/settings/test.py

```diff
@@ -72,3 +72,9 @@
 
 MIDDLEWARE_RETRIES = 10
 MIDDLEWARE_MIDDLEWARE_RETRY_SLEEPTIME = 3
+
+# During testing, if mocking isn't done right, we never want to
+# accidentally send data to Google Analytics.
+GOOGLE_ANALYTICS_API_URL = 'https://fake.guugle-analytics.fake/collect'
```

Comment:
This is a nit of mine because it's bit me several times.

In case anyone doesn't know and is reading along, there are a handful of top-level domains that are reserved for specific purposes like documenting and testing. Using these appropriately helps when reading the relevant code since the reader knows exactly what the thing is. Further, you can't accidentally have the code work with unintended consequences.

* http://www.iana.org/domains/reserved
* http://tools.ietf.org/html/rfc2606

Fun thing I just discovered: if you type "example.com" in Firefox, it goes to a nice page that talks about this.